Makefile

```diff
@@ -1,7 +1,8 @@
+CUPS_ETC_DIR?=/etc/cups
 CUPS_PPD_DIR?=/usr/share/cups/model
 CUPS_BACKEND_DIR?=/usr/lib/cups/backend
 PS2PDF?=/usr/bin/epstopdf
-MAILX?=/usr/bin/mailx
+MAILX?=$(CUPS_BACKEND_DIR)/mailx-wrapper.sh
```

Comment:
I'd like the default mailx to remain /usr/bin/mailx. Cups filters are fragile enough as is, so the default should remain the most simple codepath. This variable is for the install checks, so I'd just add an extra install check for the wrapper script.